src/libraries/System.Net.Sockets/src/System/Net/Sockets/MulticastOption.cs

```diff
@@ -67,6 +67,11 @@ public IPAddress Group
             }
             set
```

Comment:
Since this API did not have triple slash comments, and is already documented in MS Docs, this new exception must be added here:

https://github.com/dotnet/dotnet-api-docs/blob/f37171b1ee7912fad473703ce48a91f96df382b8/xml/System.Net.Sockets/MulticastOption.xml#L273